includes/Admin.php

```diff
@@ -174,6 +174,14 @@ public function prefill_post_title( $title ) {
 			return $title;
 		}
 
-		return (string) get_the_title( $post_id );
+		$post = get_post( $post_id );
```

Comment:
Why this every not be a `WP_Post`? Surely the above line `Story_Post_Type::POST_TYPE_SLUG !== get_post_type( $post_id )` would check this?